Makefile

```diff
@@ -65,8 +65,13 @@ init: $(foreach p,$(REACTION_PROJECTS),init-$(p)) post-system-start
 ###############################################################################
 ### Targets to verify Github is configured correctly.
 ###############################################################################
-ssh-public-key: ~/.ssh/id_rsa.pub
-	@echo "SSH public key verified at ~/.ssh/id_rsa.pub"
+ssh-public-key:
+	@if [ -z $$(find "${HOME}/.ssh" -name 'id_*.pub') ]; then \
+		echo "No ssh public key found. Generating one via ssh-keygen."; \
+		ssh-keygen; \
+	else \
+		echo "ssh public key verified"; \
+	fi
```

Comment:
It might be best to avoid checking for the existing of an SSH at any given filepath? A validation of `ssh -T git@github.com` would succeed for fail based on SSH configuration, regardless of key location.

Optionally, it might be best to bail and let the user fix rather than auto create a key? We could point to the Github SSH help pages. They're user friendly.